apps/tests/test_app_view_forbidden.py

```diff
@@ -11,9 +11,7 @@ class AppsViewForbidden(TestCase):
     def setUp(self):
         user = User.objects.create_user("foo", "foo@test.com", "bar")
 
-        _ = Project.objects.create_project(
-            name="test-perm", owner=user, description="", repository=""
-        )
+        _ = Project.objects.create_project(name="test-perm", owner=user, description="", repository="")
```

Comment:
I think there is no need for this assignment. If for debug only..

Reply:
In this PR I focused on only activating the code checking tools and their suggested reformatting to make them happy. We can make other changes as part of future PR:s to keep them more focused I think.